compress_jointcal_cfht_test_data.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
```

Comment:
Doesn't the shebang need to be hacked at build time to work on modern Mac systems?